README.rst

```diff
@@ -75,17 +75,19 @@ Basic Usage
          8: ['Brand X', '98', 'Internal', 'Hourly', 60]
          9: ['Brand Y', '2000', 'Modem', 'Contr.', 60]
         10: ['Brand Y', 'NT', 'Modem', 'Salaried', 60]
-        11: ['Brand Y', 'XP', 'Modem', 'Part-Time', 60]
-        12: ['Brand Y', '2000', 'Modem', 'Hourly', 30]
+        11: ['Brand X', 'XP', 'Internal', 'Part-Time', 6]
+        12: ['Brand X', '2000', 'Internal', 'Hourly', 30]
         13: ['Brand Y', '98', 'Modem', 'Contr.', 15]
-        14: ['Brand Y', 'XP', 'Modem', 'Salaried', 15]
-        15: ['Brand Y', 'NT', 'Modem', 'Part-Time', 15]
-        16: ['Brand Y', 'XP', 'Modem', 'Part-Time', 30]
-        17: ['Brand Y', '98', 'Modem', 'Part-Time', 6]
+        14: ['Brand Y', 'XP', 'Modem', 'Salaried', 10]
+        15: ['Brand X', 'NT', 'Internal', 'Part-Time', 15]
+        16: ['Brand X', 'XP', 'Internal', 'Part-Time', 30]
+        17: ['Brand Y', '98', 'Modem', 'Part-Time', 60]
         18: ['Brand Y', '2000', 'Modem', 'Salaried', 6]
-        19: ['Brand Y', '98', 'Modem', 'Salaried', 10]
-        20: ['Brand Y', 'XP', 'Modem', 'Contr.', 6]
-        21: ['Brand Y', 'NT', 'Modem', 'Hourly', 10]
+        19: ['Brand X', 'NT', 'Internal', 'Contr.', 10]
+        20: ['Brand X', '98', 'Internal', 'Hourly', 10]
+        21: ['Brand Y', 'XP', 'Modem', 'Contr.', 6]
+        22: ['Brand Y', 'XP', 'Modem', 'Salaried', 60]
+        23: ['Brand X', 'NT', 'Internal', 'Salaried', 15]
```

Comment:
Non-filtered generations now also have different outputs due to fix of uneven distribution of parameters in the resulting set.

In this example, `'Modem'` were used more than `'Internal'`. Now they are distributed evenly.